File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float dollars;
    do
    {
        dollars = get_float("Change owed in dollars?\n"); //prompt owed dollars value
    }
    while (dollars < 0);

    int cents = round(dollars * 100);
    int coins = 0;

    for (; cents >= 25; cents = cents - 25)
    {
        coins++;
    }

    for (; cents >= 10; cents = cents - 10)
    {
        coins++;
    }

    for (; cents >= 5; cents = cents - 5)
    {
        coins++;
    }

    for (; cents >= 1; cents = cents - 1)
    {
        coins++;
    }

    printf("%i", coins);
}

File: pset6/cash/cash.py
from cs50 import get_float

while True:
    dollars = get_float("Change owed in dollars?")
    if dollars >= 0:
        break

cents = round(dollars * 100)
coins = 0

if cents >= 25:
    while True:
        cents -= 25
        coins += 1
        if cents < 25:
            break

if cents >= 10:
    while True:
        cents -= 10
        coins += 1
        if cents < 10:
            break

if cents >= 5:
    while True:
        cents -= 5
        coins += 1
        if cents < 5:
            break

if cents >= 1:
    while True:
        cents -= 1
        coins += 1
        if cents < 1:
            break

print(coins)